packages/context/src/objects/objects.ts

```diff
@@ -323,9 +332,9 @@ export type ActionInput = {
         when?: string;
         eq?: string | number | boolean;
     };
-    property?: {
-        id: string;
-    };
+    property?: InputParameter;
+    viewLayout?: ViewLayoutEntityReference;
+    documentMetadata?: DocumentMetadata;
```

Comment:
Is this the legacy `inputProperties`?  Are we still maintaining this?

Reply:
Yes it is legacy `inputProperties`. 
I am using this type `ActionInput` instead of the one in Configuration repo. The one called FormIOComponent is in the Configuration repo is removed. 
There are too many duplicated types between this repo and configuration repo. So I am removing all those duplicated ones in Configuration repo to use the ones in this repo as a single source of truth.

Reply:
Also, we are using `ActionInput` type when we convert form to formio component in Widget repo.